src/view/RecoExplorerApp.py

```diff
@@ -128,9 +130,10 @@ def __init__(
     def set_c2c_model_definitions(self):
         models = self.config[constants.MODEL_CONFIG_C2C][constants.MODEL_TYPE_C2C]
         for model in models.keys():
```

Comment:
I think it would be more concise to iterate over the `model.items()` and work with the key and value directly, so you dont need to access the value at every step